### PR TITLE
BO: Improved HelperOptions for textarea field

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | As described in commits titles |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | no |
| How to test? | create new field in back-office section which uses HelperOptions and use textarea field with autoload_rte param set to true |
| Sponsor company | impSolutions

nice_textarea

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
@vTerenti what do you think ?

**Comment by Shudrum** [comment]
Hi @kpodemski!

Thank you for your contribution, but our commits norms haven't changed this way ^^

Please follow this guideline: http://build.prestashop.com/news/prestashop-coding-standards/#commits--pull-requests

Thank you!

**Comment by kpodemski** [comment]
@Shudrum it will be good enought to simply add `BO: ` prefix to both of them?

**Comment by mickaelandrieu** [comment]
@kpodemski yes :)

**Comment by kpodemski** [comment]
@mickaelandrieu done :)

**Comment by Shudrum** [comment]
👍

**Comment by mickaelandrieu** [comment]
@kpodemski I don't know how to test it, can you explain me the interest of this contribution and how I can check it please ?

**Comment by kpodemski** [comment]
Add `textarea` field on for example MaintenanceController where HelperOptions is with: `autoload_rte => true`

File: classes/helper/HelperOptions.php [hunk 1]
```diff
@@ -103,7 +103,19 @@ public function generateOptions($option_list)
                 }
 
                 if ($field['type'] == 'texarea' || $field['type'] == 'textareaLang') {
```
Comment (mickaelandrieu) [inline classes/helper/HelperOptions.php]: would you mind to fix this bug please ? <s>texarea</s> => `textarea`

**Comment by mickaelandrieu** [comment]
Works as expected, except for 1 bug from our side ...

**Comment by kpodemski** [comment]
Done

**Comment by mickaelandrieu** [comment]
BO: Fixed typo and I merge, for real :heart_eyes:

**Comment by kpodemski** [comment]
@mickaelandrieu done ^^ I see that GitHub has some issues now

**Comment by mickaelandrieu** [comment]
Thank you for your patience ;)

**Pull request merged**